interfaces/xenmgr_vm.xml

```diff
@@ -466,36 +461,9 @@
       <tp:docstring>Extra ioemu arguments separated by semicolon.</tp:docstring>
     </property>
 
-    <property name="crypto-key-dirs" type="s" access="readwrite">
-      <tp:docstring>Comma-separated list of disk encryption keys directories.</tp:docstring>
-    </property>
-
-    <property name="dependencies" type="ao" access="read">
```

Comment:
I regularly use this property and the next 4. Please do not remove them.

Reply:
These interfaces have no backend implementation on the unrestricted interface.  They're only implemented on com.citrix.xenclient.xenmgr.vm.  If you grep through the rest of xenmgr_vm.xml and look in manager/xenmgr/XenMgr/Expose/VmObject.hs, you'll see what I mean.

Reply:
Oh, those were duplicates, understood, thanks!